doc/src/installation.rst

```diff
@@ -10,85 +10,55 @@ including Apple OS X, but they are not officially tested and supported.
 Third-Party Software Packages
 -----------------------------
 
-**Python 2** ``>=`` **2.6** is required. **Python 2** ``>=`` **2.7.9** is
-recommended for the best security. `Python <https://python.org/>`_ 2 should
-already be installed in your Linux system.
+Requirements:
 
-For Cylc's HTTPS communications layer:
+- Python 3.6+
 
-- `OpenSSL <https://www.openssl.org/>`_
-- `pyOpenSSL <http://www.pyopenssl.org/>`_
-- `python-requests <http://docs.python-requests.org/>`_
-- **python-urllib3** - should be bundled with python-requests
+  - `python-jose <https://pypi.org/project/python-jose/>`_
+  - `zmq <https://pypi.org/project/zmq/>`_
+  - `colorama <https://pypi.org/project/colorama/>`_
 
-The following packages are highly recommended, but are technically optional as
-you can construct and run suites without dependency graph visualisation or
-the Cylc GUIs:
-
-- `PyGTK <http://www.pygtk.org>`_ - GUI toolkit.
-
-  .. note::
-
-     PyGTK typically comes with your system Python. It is allegedly quite
-     difficult to install if you need to do so for another Python version.
-
-- `Graphviz <http://www.graphviz.org>`_ - graph layout engine (tested 2.36.0)
-- `Pygraphviz <http://pygraphviz.github.io/>`_ - Python Graphviz interface
-  (tested 1.2). To build this you may need some *devel* packages too:
-  
-  - python-devel
-  - graphviz-devel
-
-The Cylc Review service does not need any additional packages.
-
-The following packages are necessary for running all the tests in Cylc:
+The following packages are necessary for running tests in Cylc:
 
 - `mock <https://mock.readthedocs.io>`_
```

Comment:
Is `mock` now part of standard library?

Reply:
Pre python 3 it was a standalone library. Post Python 3 I believe it was incorporated in `unittest` (not sure if 3.+ or if it was 3.3, 3.4, etc).

Reply:
Its doc says new in 3.3. If we are going for 3.6+ or 3.7+, we should no longer have to worry about it being an external dependency. (I don't think anyone would use 3.3 any more!? Even our brand new ancient RHEL7 desktop is on 3.4!)